site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -39,13 +40,27 @@ const Planner: FC = () => {
   const [missingPrerequisites, setMissingPrerequisites] = useState(new Set<string>());
 
   useEffect(() => {
-    // if is first render, load from local storage
-    if (isFirstRenderer) {
+    // stringify current roadmap
+    const roadmapStr = JSON.stringify({
+      planner: collapsePlanner(data),
+      transfers: transfers,
+    });
+
+    // stringified value of an empty roadmap
+    const emptyRoadmap =
+      '{"planner":[{"startYear":2024,"name":"Year 1","quarters":[{"name":"fall","courses":[]},{"name":"winter","courses":[]},{"name":"spring","courses":[]}]}],"transfers":[]}';
```

Comment:
I would rather construct this as a SavedRoadmap object first then stringify. That way we get the type safety and can avoid any bugs down the road when the schema changes (e.g. when we add multiple planner support).

```suggestion
    const emptyRoadmap = JSON.stringify({
      planner: [
        {
          startYear: 2024,
          name: 'Year 1',
          quarters: [
            { name: 'fall', courses: [] },
            { name: 'winter', courses: [] },
            { name: 'spring', courses: [] },
          ],
        },
      ],
      transfers: [],
    } as SavedRoadmap);
```